src/App.jsx

```diff
@@ -1,31 +1,88 @@
-import React, { useState, useEffect } from 'react';
-import { getAllData } from './util/index';
+import React from "react";
+import {BrowserRouter as Router, Route, Routes, Link, useLocation} from "react-router-dom";
+import { GlobeEuropeAfricaIcon, Bars3Icon } from "@heroicons/react/24/outline";
+import MainPage from "./pages/MainPage";
+import About from "./pages/About";
+import Opportunities from "./pages/Opportunities";
+import OpportunityDetails from "./pages/OpportunityDetails";
+import ContactUs from "./pages/ContactUs";
+import LogIn from "./pages/LogIn";
 
-const URL = 'http://localhost:8000/api/v1/';
+function Navigation() {
+  const location = useLocation();
 
-function App() {
-  
-  const [message, setMessage] = useState(''); 
-
-  useEffect(() => {
-
-    (async () => {
-      const myData = await getAllData(URL)
-      setMessage(myData.data);
-    })();
-      
-    return () => {
-      console.log('unmounting');
-    }
-
-  }, []);
+  const navigation = [
+    { name: "Opportunities", href: "/opportunities" },
+    { name: "About", href: "/about" },
+    { name: "Contact us", href: "/contact-us" },
+    { name: "Log In", href: "/login" },
+  ];
 
   return (
     <>
-      <h1>{message}</h1>
+      <div className="md:hidden">
+        <Bars3Icon aria-hidden="true" className="h-6 w-6" />
```

Comment:
Looks like the navigation bar on mobile isn't setup yet. Can you add a comment here saying that, and create a new Jira ticket to track as future work?

Reply:
Done!